snmp/datadog_checks/snmp/data/default_profiles/avaya-nortel-ethernet-routing-switch.yaml

```diff
@@ -0,0 +1,73 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - avaya.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.45.3.71.10 # 4526T
+
+metrics:
+  - MIB: S5-CHASSIS-MIB
```

Comment:
Any idea what's the meaning of `S5` ?

Reply:
Seems to mean `series5000`
Added `avaya` prefix for clarity